addons/website_sale_product_configurator/static/src/js/website_sale_options.js

```diff
@@ -0,0 +1,102 @@
+odoo.define('website_sale_options.website_sale', function (require) {
+'use strict';
+
+var core = require('web.core');
+var publicWidget = require('web.public.widget');
+var OptionalProductsModal = require('sale_product_configurator.OptionalProductsModal');
+require('website_sale.website_sale');
+
+var _t = core._t;
+
+publicWidget.registry.WebsiteSale.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    _onProductReady: function () {
+        this.optionalProductsModal = new OptionalProductsModal(this.$form, {
+            rootProduct: this.rootProduct,
+            isWebsite: true,
+            okButtonText: _t('Proceed to Checkout'),
+            cancelButtonText: _t('Continue Shopping'),
+            title: _t('Add to cart'),
+            context: this._getContext(),
+        }).open();
+
+        this.optionalProductsModal.on('options_empty', null, this._submitForm.bind(this));
+        this.optionalProductsModal.on('update_quantity', null, this._onOptionsUpdateQuantity.bind(this));
+        this.optionalProductsModal.on('confirm', null, this._onModalConfirm.bind(this));
```

Comment:
might be written ..., this._onModalSubmit.bind(this, false));
and  ..(this, true));

Reply:
Done, that allows some nice code cleaning, thanks.